tensorboard/webapp/metrics/store/metrics_reducers.ts

```diff
@@ -941,8 +941,23 @@ const reducer = createReducer(
     };
   }),
   on(actions.selectTimeEnableToggled, (state) => {
+    let nextCardStepIndexMap = {...state.cardStepIndex};
+    // Updates cardStepIndex only when toggle to enable selectedTime.
+    if (!state.selectTimeEnabled) {
```

Comment:
This is a little confusing from a readability perspective: It reads like "if select time is not enabled" but in practice it actually means "if select time is enabled".

Maybe define: `const selectTimeEnabled = !state.selectTimeEnabled;` and use it here and on L961.

Reply:
Good suggestion. Updated.